airflow/providers/amazon/aws/operators/appflow.py

```diff
@@ -0,0 +1,545 @@
+    :param aws_conn_id: aws connection to use
+    :param region: aws region to use
+    """
+
+    template_fields = ("dt",)
+
+    def __init__(
+        self,
+        source: str,
+        name: str,
+        source_field: str,
+        dt: str,
+        poll_interval: int = 20,
+        aws_conn_id: Optional[str] = "aws_default",
+        region: Optional[str] = None,
+        **kwargs,
+    ) -> None:
+        if not dt:
+            raise AirflowException("The dt argument is mandatory for AppflowRunAfterOperator!")
+        if source not in {"salesforce", "zendesk"}:
+            raise AirflowException(f"Source {source} is not supported for AppflowRunAfterOperator!")
+        super().__init__(
+            source=source,
+            name=name,
+            flow_update=True,
+            source_field=source_field,
+            dt=dt,
+            poll_interval=poll_interval,
+            aws_conn_id=aws_conn_id,
+            region=region,
+            **kwargs,
+        )
+
+    def _add_filter(self, connector_type: str, tasks: List["TaskTypeDef"]) -> None:
+        if not self.dt_parsed:
+            raise AirflowException(f"Invalid dt argument parser value: {self.dt_parsed}")
+        if not self.source_field:
+            raise AirflowException(f"Invalid source_field argument value: {self.source_field}")
+        filter_task: "TaskTypeDef" = {
+            "taskType": "Filter",
+            "connectorOperator": {connector_type: "GREATER_THAN"},  # type: ignore
+            "sourceFields": [self.source_field],
+            "taskProperties": {
+                "DATA_TYPE": "datetime",
+                "VALUE": str(datetime_to_epoch_ms(self.dt_parsed)),
+            },  # NOT inclusive
+        }
+        tasks.append(filter_task)
+
+
+class AppflowRunDailyOperator(AppflowBaseOperator):
+    """
+    Execute a Appflow run after updating the filters to select only a single day.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:AppflowRunDailyOperator`
+
+    :param source: The source name (e.g. salesforce)
+    :param name: The flow name
+    :param source_field: The field name to apply filters
+    :param dt: The date value (or template) to be used in filters.
+    :param poll_interval: how often in seconds to check the query status
+    :param aws_conn_id: aws connection to use
+    :param region: aws region to use
+    """
+
+    template_fields = ("dt",)
+
+    def __init__(
+        self,
+        source: str,
+        name: str,
+        source_field: str,
+        dt: str,
+        poll_interval: int = 20,
+        aws_conn_id: Optional[str] = "aws_default",
+        region: Optional[str] = None,
+        **kwargs,
+    ) -> None:
+        if not dt:
+            raise AirflowException("The dt argument is mandatory for AppflowRunDailyOperator!")
+        if source not in {"salesforce"}:
```

Comment:
```suggestion
        if source != "salesforce":
```
Also same suggestion as above instances too re: the exception type and message formatting. It might be worth creating a global variable for the exception message since it's repeated often.

Reply:
Done!